aziotctl/config/unix/template.toml

```diff
@@ -6,6 +6,14 @@
 #
 # hostname = "my-device"
 
+# ==============================================================================
+# Local Gateway Hostname
+# ==============================================================================
+#
+# If this device is running in a nested deployment, uncomment the next line to
+# set the local gateway hostname of this device.
+#
+# local_gateway_hostname = "my-parent-device"
```

Comment:
@daprilik - I thought this was going to be called parent_hostname. 
Also, should this not be in identityd config as well?